<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamTeacher</class>
 <widget class="QWidget" name="ExamTeacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="examsTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>1401</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学年</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>学期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课程</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>老师</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>结束时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>阅卷</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>3</y>
     <width>851</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>发布</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="academicYearLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="semesterLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="courseLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="teacherLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="dataLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="endLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="searchButton">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
